server side/router/profile.router.js

```diff
@@ -0,0 +1,9 @@
+import express from 'express'
+import {getAllProfiles,getProfileById,createProfile,deleteProfile,updateProfile} from '../controllers/profileController'
+const router=express.Router();
+router.get('/getAllProfiles',getAllProfiles);
+router.get('/getProfileById/:id',getProfileById);
+router.post('/createProfile',createProfile);
+router.delete('/deleteProfile/:id',deleteProfile);
+router.put('/updateProfile/:id',updateProfile);
```

Comment:
remove the create-update-delete words, look in the user router for the correct way to do it.